### DISCUSSION TITLE
Supabase filter requires an object variable.

### DISCUSSION THREAD

**Discussion by MachariaK** [supabase/supabase#20061] [Questions] [answered]
In Supabase Realtime, I aim to implement a filter that depends on either of two fields. Is it possible to use an OR condition in the PostgresChangeFilter for the following scenarios?
```
PostgresChangeFilter(
  type: PostgresChangeFilterType.eq,
  column: 'saved_by',
  value: broadcasterId,
),
PostgresChangeFilter(
  type: PostgresChangeFilterType.eq,
  column: 'updated_by',
  value: broadcasterId,
)
```

**Comment by GaryAustin1** [comment, marked as answer]
No.  Realtime is limited to the basic filters in the realtime docs and a single filter. https://supabase.com/docs/guides/realtime/postgres-changes#filtering-for-specific-changes
